spec/factories/users.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require 'faker'
+
+FactoryBot.define do
+  factory :user do
+    sequence(:email) { |n| Faker::Internet.email + n.to_s }
+    password { Faker::Internet.password }
```

Comment:
In the devise initializer password has a minimum length of 6. It's a good practice to develop factories that always generate a valid model. You can use the param `min_length`